deeplake/core/dataset/dataset.py

```diff
@@ -1873,6 +1873,7 @@ def pytorch(
         pad_tensors: bool = False,
         transform_kwargs: Optional[Dict[str, Any]] = None,
         decode_method: Optional[Dict[str, str]] = None,
+        cache_size: int = 32 * MB,
```

Comment:
I am curious, how did we came up with this cache size? did we do a benchmarking or is it a known constant that proved to work best?

Reply:
This was the already existing cache size. It does not matter anymore because we will hold at least one chunk in memory.